build/dockerfiles/rhel.Dockerfile

```diff
@@ -21,8 +21,8 @@ USER 0
 
 ARG BOOTSTRAP=false
 ENV BOOTSTRAP=${BOOTSTRAP}
-ARG LATEST_ONLY=false
-ENV LATEST_ONLY=${LATEST_ONLY}
+ARG USE_DIGESTS=false
+ENV USE_DIGESTS=${USE_DIGESTS}
```

Comment:
ok, I can probably use this in Brew if I overwrite with `env USE_DIGESTS=true`

Reply:
I figured there was a reason to have `ENV BOOTSTRAP` above. I can add this to the plugin registry PR as well.

Reply:
Sweet, please do.